### DISCUSSION TITLE
Support configuration cache?

### DISCUSSION THREAD

**Discussion by mochadwi** [dipien/semantic-version-gradle-plugin#25] [General]
We're currently using https://github.com/ethauvin/semver-gradle as it seems undermaintained.

Hope for the dipien/semantic-version-gradle-plugin support for configuration cache(?)

**Comment by maxirosson** [comment]
I didn't take into account configuration cache when implementing this plugin. I created an issue to tackle this topic on v1.4: https://github.com/dipien/semantic-version-gradle-plugin/issues/24

Not sure when In will have a bit of spare time to work on it.

**Comment by maxirosson** [comment]
I released v1.3.1 with some fixes, so now the plugin is not going to fail if you enable the configuration cache. 

Basically, I declared all the tasks as not compatible with configuration cache while planning a migration ([docs](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:task_opt_out)). This way, the plugin won't make clients build fail if they enable configuration cache for now.
